source/_integrations/nad.markdown

```diff
@@ -71,7 +79,7 @@ max_volume:
   default: -20
   type: integer
 sources:
-  description: A list of mappings from source to source name. Valid sources are `1 to 10`. (for `RS232` and `Telnet` types)
+  description: A list of mappings from source to source name. Valid sources are `1 to 12`. (for `RS232` and `Telnet` types)
```

Comment:
If we're changing the source range we should target next branch since that requires the parent PR in core.

Reply:
Thanks, I've now changed the PR to the next branch.